tls/vibe/stream/openssl.d

```diff
@@ -49,6 +49,8 @@ version(VibeForceALPN) enum alpn_forced = true;
 else enum alpn_forced = false;
 enum haveALPN = OPENSSL_VERSION_NUMBER >= 0x10200000 || alpn_forced;
 
+enum haveKeylog = OPENSSL_VERSION_AT_LEAST(1, 1, 1);
```

Comment:
`private` ?

Reply:
Since we optionally define the functions based on this, leaving the enum public helps determine whether we can register the callback (in the event where we don't want to use the env var, but directly assign the callback).

Otherwise, one has to use `__traits(compiles, ...)` or `is(typeof(...))`